### PR TITLE
Dependabot has issues :robot: 

### PR DESCRIPTION
**Pull request by alismx**
# DEVOPS PULL REQUEST

## Changes Proposed

- Create GitHub issues on dependabot PR creation
- Close GitHub issues on dependabot PR close

## Additional Information

- Permissions need to be explicitly set for issues and pull-requests to that the actions can write to those objects.

## Testing

-

### CODE REVIEW

File: .github/workflows/dependabotClosePRIssue.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    runs-on: ubuntu-latest
+    env:
+      GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
+    # Github action permissions: https://docs.github.com/en/actions/using-jobs/assigning-permissions-to-jobs#overview
+    permissions:
+      pull-requests: write
+    steps:
+      - uses: actions/checkout@v2
+        if: github.event.pull_request.user.login  == 'dependabot[bot]'
+      - name: Open issue if Dependabot PR
```
Comment (emmastephenson) [inline .github/workflows/dependabotClosePRIssue.yml:17]: If I'm understanding correctly, I think the name of this job should be "Close issue after Dependabot PR merged" or similar.

Comment (alismx) [reply .github/workflows/dependabotClosePRIssue.yml:17]: gah! You caught me. I'll push up a fix. Thanks, Emma!

Comment (alismx) [reply .github/workflows/dependabotClosePRIssue.yml:17]: @emmastephenson I'm going to name it "Close issue if Dependabot PR is closed" This job will close the ticket on PR close; that could be a close with a merge or a close without.

**Review by emmastephenson — APPROVED** [review]
LGTM!

**Review by mehansen — APPROVED** [review]
👍 🤖

**Pull request merged**